pkgs/development/tools/vala-language-server/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitHub
+, nix-update-script
+, meson
+, ninja
+, pkg-config
+, scdoc
+, gnome-builder
+, dash
+, gnused
+, glib
+, libgee
+, json-glib
+, jsonrpc-glib
+, vala
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vala-language-server";
+  version = "0.48.1";
+
+  src = fetchFromGitHub {
+    owner = "benwaffle";
+    repo = pname;
+    rev = version;
+    sha256 = "12k095052jkvbiyz8gzkj6w7r7p16d5m18fyikl48yvh5nln8fw0";
+  };
+
+  passthru = {
+    updateScript = nix-update-script {
+      attrPath = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    scdoc
+    # GNOME Builder Plugin
+    dash
```

Comment:
the nix sandbox has /bin/sh, so I wonder if find_program https://github.com/benwaffle/vala-language-server/blob/master/plugins/gnome-builder/meson.build#L7 considers /bin, or maybe we'd have to do
```
preBuild = ''
  # sh for gnome-builder-plugin
  export PATH=$PATH:/bin
'';

Reply:
Seems to work without it: `Program sh found: YES`

Reply:
I am not sure if I used sand-boxing tough